charts/umbrella/values.yaml

```diff
@@ -677,7 +677,7 @@ tx-data-provider:
   enabled: false
   seedTestdata: true
   backendUrl: http://{{ .Release.Name }}-dataprovider-submodelserver:8080
-  registryUrl: http://{{ .Release.Name }}-dataprovider-dtr:8080/api/v3.0
+  registryUrl: http://{{ .Release.Name }}-dataprovider-dtr:8080/api/v3
```

Comment:
I am wondering why this line is showing, because the changes are already in the branch upgrade/24.05

Reply:
i merged upgrade/24.05 into this branch. Should be gone now